### PR TITLE
[dashboard] Onboarding: new video onboarding for new workspace

### PR DESCRIPTION
**Pull request by Siddhant-K-code**
## Description

This PR adds a new video onboarding feature for new workspaces. The feature includes a YouTube video that provides an overview of Gitpod and how to create a new workspace. When the video is played, an event is tracked using the `trackVideoClick` function.

When someone clicks on YT video, it would trigger a segment event with the following payload as event `video_clicked`:

```json
{
  "anonymousId": "895ajdj47-14yy-4bab-b113-xxxxxx",
  "event": "video_clicked",
  "properties": {
    "context": "create-new-workspace",
    "path": "/workspaces"
  }
}
```

**ACTION REQUIRED:** Update YouTube Video

## Related Issue(s)

Fixes OPM-655

## How to test

- Open preview environment.
- Complete initial steps
- After completion, you will see a video with updated copy text on `/workspaces`

#### Preview status

<p>Gitpod was successfully deployed to your preview environment.</p>
<ul>
	<li><b>🏷️ Name</b> - siddhant-o6097563f27</li>
	<li><b>🔗 URL</b> - <a href="https://siddhant-o6097563f27.preview.gitpod-dev.com/workspaces" target="_blank">siddhant-o6097563f27.preview.gitpod-dev.com/workspaces</a>.</li>
	<li><b>📚 Documentation</b> - See our <a href="https://www.notion.so/gitpod/6debd359591b43688b52f76329d04010#7c1ce80ab31a41e29eff2735e38eec39" target="_blank">internal documentation</a> for information on how to interact with your preview environment.</li>
	<li><b>📦 Version</b> - siddhant-opm-655-add-video-to-dashboard-when-onboarding-gha.26994</li>
	<li><b>🗒️ Logs</b> - <a href="https://console.cloud.google.com/logs/query;query=jsonPayload.kubernetes.host%3D%22preview-siddhant-o6097563f27%22%0A%0A--%20Filter%20on%20service:%0A--%20jsonPayload.serviceContext.service%3D%22ws-manager-mk2%22%0A;duration=P1D?project=gitpod-dev-preview" target="_blank">GCP Logs Explorer</a></li>
</ul>

## Build Options

<details>
<summary>Build</summary>

- [ ] /werft with-werft
      Run the build with werft instead of GHA
- [ ] leeway-no-cache
- [ ] /werft no-test
      Run Leeway with `--dont-test`
</details>

<details>
<summary>Publish</summary>

- [ ] /werft publish-to-npm
- [ ] /werft publish-to-jb-marketplace
</details>

<details>
<summary>Installer</summary>

- [x] analytics=segment
- [ ] with-dedicated-emulation
- [ ] workspace-feature-flags
  Add desired feature flags to the end of the line above, space separated
</details>

<details>
<summary>Preview Environment / Integration Tests</summary>

- [ ] /werft with-local-preview
      If enabled this will build `install/preview`
- [x] /werft with-preview
- [ ] /werft with-large-vm
- [x] /werft with-gce-vm
      If enabled this will create the environment on GCE infra
- [x] /werft preemptible
      Saves cost. Untick this only if you're really sure you need a non-preemtible machine.
- [ ] with-integration-tests=all
      Valid options are `all`, `workspace`, `webapp`, `ide`, `jetbrains`, `vscode`, `ssh`. If enabled, `with-preview` and `with-large-vm` will be enabled.
- [ ] with-monitoring
</details>

/hold

### CODE REVIEW

File: components/dashboard/src/workspaces/EmptyWorkspacesContent.tsx [hunk 1]
```diff
@@ -4,34 +4,71 @@
...
-                            <span className="opacity-60 hidden md:inline">{StartWorkspaceModalKeyBinding}</span>
-                        </LinkButton>
-                    </span>
+            <div className="px-6 py-3 flex flex-row items-center justify-center space-x-16">
+                <div>
+                    <iframe
+                        id="gitpod-video"
+                        width="535"
+                        height="307"
+                        src="https://www.youtube.com/embed/R6FQ39sitAQ?enablejsapi=1&modestbranding=1&rel=0&controls=1&showinfo=0&fs=1"
```
Comment (Siddhant-K-code) [inline components/dashboard/src/workspaces/EmptyWorkspacesContent.tsx:52]: **TODO** (@loujaybee/@philschatz) - Update YT video

File: components/dashboard/src/workspaces/EmptyWorkspacesContent.tsx [hunk 2]
```diff
@@ -4,34 +4,71 @@
...
+                        title="YouTube - Gitpod in 120 seconds"
```
Comment (phimae) [inline components/dashboard/src/workspaces/EmptyWorkspacesContent.tsx:53]: @Siddhant-K-code if we remove this line, would it remove the title from the thumbnail of the video? Imo it's best to have this look as clean as possible, almost as if we had just embedded it ourselves. 

image

Comment (Siddhant-K-code) [reply components/dashboard/src/workspaces/EmptyWorkspacesContent.tsx:53]: > if we remove this line, would it remove the title from the thumbnail of the video?

Nope, It's a YT Embed styles

**Review by svenefftinge — APPROVED** [review]
LGTM

**Comment by Siddhant-K-code** [comment]
**FYI**: https://github.com/gitpod-io/gitpod/pull/19991 should be merged before this PR

File: components/dashboard/src/workspaces/EmptyWorkspacesContent.tsx (hunk 1 above)
Comment (Siddhant-K-code) [reply components/dashboard/src/workspaces/EmptyWorkspacesContent.tsx]: Updated ✅

**Pull request merged**